storage/innobase/log/log0log.cc

```diff
@@ -559,7 +559,7 @@ that the proper size of the log buffer should be a power of two.
 @param[out]     log             redo log */
 static void log_calc_buf_size(log_t &log);
 
-uint32_t log_detected_format = UINT32_MAX;
+Log_format log_detected_format = Log_format::LEGACY;
```

Comment:
Do we use this log_detected_format at all? And the redo log created during prepare will have the 8.0.30 tag which are applied and later removed.

Reply:
we use log_detected_format for mach_write_compressed and few other methods in mach0data.ic

Reply:
Hmm, we use it for mach_read_compressed(). Good point @rahulmalik87

Reply:
and  mach_write_compressed() incase we write redo log during trx rollback etc.